velox/exec/tests/SpillTest.cpp

```diff
@@ -42,46 +43,156 @@ class SpillTest : public testing::Test,
 
 TEST_F(SpillTest, spillState) {
   auto tempDirectory = exec::test::TempDirectoryPath::create();
+  const std::string kSpillPath = tempDirectory->path + "/test";
+  const int kNumPartitions = 2;
+  // Set the target file size to a large value to avoid new file creation on
+  // each batch write.
+  const int kTargetFileSize = 1'000'000'000;
+  std::shared_ptr<FileSystem> fs;
+  std::vector<std::string> spilledFiles;
+  {
+    // We make a state that has 2 partitions, each with its own file
+    // list. We write 10 sorted vectors in each partition. The vectors
+    // have the ith element = i * 20 + sequence, where sequence is the
```

Comment:
I find it hard to understand the data pattern here. The code below suggests it is i * 10 + sequence, not i * 20 + sequence.

`[&](auto row) { return row * 10 + batch; }`